extensions/wikia/EditPageLayout/js/plugins/PageControls.js

```diff
@@ -436,15 +401,16 @@
 		renderPreview: function (extraData, type) {
 			var self = this;
 
-			require([ 'wikia.fluidlayout' ], function (fluidlayout) {
+			require([ 'wikia.breakpointsLayout' ], function (breakpointsLayout) {
 				var previewPadding = 22, // + 2px for borders
-					articleWidth = 660, width = articleWidth + (self.isGridLayout ? 30 : 0
-						), railBreakPoint = fluidlayout.getBreakpointSmall(), config = self.editor.config;
+					articleWidth = 660,
+					width = articleWidth + (self.isGridLayout ? 30 : 0),
```

Comment:
Can we use "constants" from `breakpointsLayout` here? I think this one should be `BREAKPOINT_LARGE_PADDING_WIDTH`